### PR TITLE
[Form] Enhance some FormRegistry deprecation messages

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

File: src/Symfony/Component/Form/FormRegistry.php [hunk 1]
```diff
@@ -120,7 +120,7 @@ private function resolveAndAddType(FormTypeInterface $type)
         $hasCustomName = $name !== $fqcn;
 
         if ($parentType instanceof FormTypeInterface) {
-            @trigger_error('Returning a FormTypeInterface from FormTypeInterface::getParent() is deprecated since version 2.8 and will be removed in 3.0.', E_USER_DEPRECATED);
+            @trigger_error(sprintf('Returning a FormTypeInterface from %s::getParent() is deprecated since version 2.8 and will be removed in 3.0.', $fqcn), E_USER_DEPRECATED);
```
Comment (Tobion) [inline src/Symfony/Component/Form/FormRegistry.php]: You could add `Return the fully-qualified type class name instead.` there as well.

Comment (nicolas-grekas) [inline src/Symfony/Component/Form/FormRegistry.php]: good idea, done

**Comment by stof** [comment]
:+1:

**Comment by Tobion** [comment]
:+1:

**Comment by Tobion** [comment]
Thank you @nicolas-grekas.

**Pull request merged**